01-rstudio-intro.Rmd

```diff
@@ -330,8 +361,8 @@ ls()
 > instead
 >
 
-Note here that we didn't given any arguments to `ls`, but we still needed to give
-the brackets to tell R to call the function.
+Note here that we didn't given any arguments to `ls`, but we still
+needed to give the brackets to tell R to call the function.
```

Comment:
I'd be happy to fix the language around parentheses, brackets, and braces,
and resubmit.

On Mon, Aug 17, 2015 at 12:00 PM, Robert McDonald <rmcd1024@gmail.com>
wrote:

> On Mon, Aug 17, 2015 at 11:00 AM, Naupaka Zimmerman <
> notifications@github.com> wrote:
>
>> In 01-rstudio-intro.Rmd
>> <https://github.com/swcarpentry/r-novice-gapminder/pull/56#discussion_r37203407>
>> :
>>
>> > @@ -330,8 +361,8 @@ ls()
>> >  > instead
>> >  >
>> >
>> > -Note here that we didn't given any arguments to `ls`, but we still needed to give
>> > -the brackets to tell R to call the function.
>> > +Note here that we didn't given any arguments to `ls`, but we still
>> > +needed to give the brackets to tell R to call the function.
>>
>> terminology: I would call these () symbols parentheses, these []
>> brackets, and these {} braces. Is that just a regional thing?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/swcarpentry/r-novice-gapminder/pull/56/files#r37203407>
>> .
>>
> I agree about terminology. The existing document referred to () as
> brackets. I meant to change those occurrences to parentheses.
>